config/7.0/obdemo-bank/ig/scripts/groovy/ModifyPaymentIntentResponse.groovy

```diff
@@ -0,0 +1,22 @@
+SCRIPT_NAME = "[ModifyPaymentIntentResponse] - "
+logger.debug(SCRIPT_NAME + " Running...")
+
+
+next.handle(context, request).thenOnResult({ response ->
+    def responseBody = response.getEntity().getJson();
+
+    if (responseBody) {
+        try{
+            responseBody.get("Data").get("Initiation").get("DebtorAccount").remove("AccountId")
+            responseBody.Data.Initiation.DebtorAccount.AccountId
```

Comment:
That line seems that isn't doing anything, can we delete it?

Reply:
I forgot to remove it. I will delete in the next commit.